### PR TITLE
Deployed Uniswap, Pacman, Added and deployed Supermario, pgAdmin4, MongoDB, Caddy WebServer app, ID: 81ow3i7lnhnivs9

### PR DESCRIPTION
**Pull request by davaymne**
Deployed Uniswap app, ID: 81ow3i7lnhnivs9
http://mt60bmuka993hbirvq0fis7ubk.provider4.akashdev.net/#/swap

{
  "services": {
    "web": {
      "name": "web",
      "available": 1,
      "total": 1,
      "uris": [
        "mt60bmuka993hbirvq0fis7ubk.provider4.akashdev.net"
      ],
      "observed-generation": 0,
      "replicas": 0,
      "updated-replicas": 0,
      "ready-replicas": 0,
      "available-replicas": 0
    }
  },
  "forwarded-ports": {}
}

### CODE REVIEW

**Comment by davaymne** [comment]
Screenshot 2020-12-07 at 23 20 49

**Comment by boz** [comment]
@davaymne, please read the instructions and look at the other entries.

Did you deploy an sdl from the entry that is already [here](https://github.com/ovrclk/awesome-akash/tree/master/uniswap) for uniswap?

**Comment by davaymne** [comment]
> @davaymne, please read the instructions and look at the other entries.
> 
> Did you deploy an sdl from the entry that is already [here](https://github.com/ovrclk/awesome-akash/tree/master/uniswap) for uniswap?
yes, I deployed from SDL which is in the list.
I followed with Open-ended Deployments (Deployed an application from the Awesome Akash List). Should I PR any other way?

**Comment by davaymne** [comment]
Screenshot 2020-12-08 at 00 18 53

**Comment by davaymne** [comment]
Screenshot 2020-12-08 at 01 17 46

File: README.md [hunk 1]
```diff
@@ -35,5 +35,35 @@ Awesome DeFi apps you can deploy on Akash
...
+      "observed-generation": 0,
+      "replicas": 0,
+      "updated-replicas": 0,
+      "ready-replicas": 0,
+      "available-replicas": 0
+    }
+  },
+  "forwarded-ports": {}
+}
+
```
Comment (boz) [inline README.md:69]: This should not be here.

File: uniswap/81ow3i7lnhnivs9.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"lease_id":{"owner":"akash12myahmse3l8cqr2tl9dl2n6vgsnmlcanw6l9er","dseq":"146505","gseq":1,"oseq":1,"provider":"akash174hxdpuxsuys9qkauaf57ym5j8dm4secnz6jd7"},"state":"active","price":{"denom":"uakt","amount":"1219"}}
```
Comment (boz) [inline uniswap/81ow3i7lnhnivs9.json:1]: This file should not be here.

**Review by boz — CHANGES_REQUESTED** [review]
Please see #19 , #22 for what is required in the PR.

**Pull request merged**